heimdall2/values.yaml

```diff
@@ -2,6 +2,12 @@
 # https://github.com/mitre/heimdall2/wiki/Environment-Variables-Configuration
 # apps/backend/.env-example
 
+# Heimdall2 Helm Chart Values File
+# This file is used to set the default values for the Heimdall2 Helm chart.
+# Any values that are not set will use the defaults specified in the chart.
+# The following values are required to be set in order for Heimdall2 to function properly:
+# NODE_ENV, JWT_SECRET, EXTERNAL_URL
```

Comment:
I dunno if it's worthwhile stating this since you honestly need to be setting a lot more values as well in the typical case.  Ex. while you could let it just autochoose a dbname, you should probably be setting one.